core/node/main/config.hpp

```diff
@@ -20,7 +20,7 @@ namespace fc::node {
   struct Config {
     boost::filesystem::path repo_path;
     spdlog::level::level_enum log_level;
-    int port = 0;
+    int port = 2000;
```

Comment:
Is this because after libp2p update we get empty address list?
Auto-assigned ports are better.

Reply:
Yes, because libp2p doesn't return listen addresses. 2000 is a default port in Lotus.